astar_lookahead_pub_node:
  ros__parameters:
    lookahead_distance: 4.5
    lookahead_marker_topic: "astar_lookahead_marker"
    is_antiClockwise: False 
    csv_path: "/home/ubuntu/giu_f1tenth_ws/software/src/planning/trajectory_planning/path/cluj_napoca_berlin_out.csv"